src/PatientChecking/PatientChecking/Views/Dashboard/Home.cshtml

```diff
@@ -2,4 +2,67 @@
     ViewData["Title"] = "Dashboard";
     ViewData["Page"] = "Dashboard";
 }
-<p>This is dashboard</p>
+@section css {
+    <link href="@Url.Content("~/css/Dashboard.css")" rel="stylesheet" type="text/css" />
+}
+<script>
+    (function () {
+        $('#loading-div-background').show();
+        
+        $.ajax({
+            type: 'GET',
+            url: 'https://localhost:5001/Dashboard/getDashBoard',
+            success: function (data) {
```

Comment:
We should handle error case. Alert user to reload the page and try again.

Reply:
added case error when calling api